druntime/Makefile

```diff
@@ -117,7 +117,6 @@ UTFLAGS:=-version=CoreUnittest -unittest -checkaction=context
 
 # Set PHOBOS_DFLAGS (for linking against Phobos)
 PHOBOS_PATH=../../phobos
-SHARED=$(if $(findstring $(OS),linux freebsd),1,)
```

Comment:
This is already set in line 79, where it is (most likely correctly) enabled on DragonFly BSD too.